### DISCUSSION TITLE
Placement Modifiers. Nested columns

### DISCUSSION THREAD

**Discussion by craftyweb** [OrchardCMS/OrchardCore#15919] [Q&A]
Hi, 

Just wondering if it's possible to do something like 
**Part:0|Info_lg-8;1|Name_lg-6;1**

It's for a Person type and I would like to create two columns (Info and Images)
I would like to place the first name and the surname on the same row within the info column but I don't know if this is possible.

Thanks

**Comment by Piedone** [comment]
I don't really understand what exactly you'd like to achieve, can you elaborate? You can have two columns, yes (but for styling that you may need to override the Content shape with your custom HTML, add custom content zones, and style it from CSS).

**Reply by craftyweb** [reply]
``Hi, I'm sorry if my intention was'nt clear enough. I'll try to specify it a bit. 

Let's say you have a Person type with the following fields

- Name: TextField
- Surname: TextField
- Email: TextField
- Picture: MediaField

When I insert the modifiers in placement I can achieve this:
```html
<div class="row">
   <div class="col-md-6">
      <!-- Default Content column -->
      <Name.TextField/>
      <Surname.TextField/>
      <Email.TextField/>
   </div>
   <div class="col-md-6">
      <!-- Media column -->
      <Picture.MediaField/>
   </div>
</div>
```

But what I would like is to place the Name and Surname field on the same row inside the Content column. Something like this:

```html
<div class="row">
   <div class="col-md-6">
      <!-- Default Content column -->
      <div class="row">
         <div class="col-md-6">
            <Name.TextField/>
         </div>
         <div class="col-md-6">
            <Surname.TextField/>
         </div>
       </div>
      <Email.TextField/>
   </div>
   <div class="col-md-6">
      <!-- Media column -->
      <Picture.MediaField/>
   </div>
</div>
```

But I don't know if something like this is possible

**Reply by Piedone** [reply]
I see. Yes, this is possible, but not with just Placement, since you also want to change the structure of the layout. You'll have to override the Content shape template, most possibly just for the given content type, see: https://docs.orchardcore.net/en/latest/reference/modules/Templates/#content-templates.
